pyfar/classes/coordinates.py

```diff
@@ -1590,6 +1590,11 @@ def find_within(
             ``'spherical_meter'``
                 distance is determined by the great-circle distance
                 expressed in meters.
+        atol : float
+            Absolute tolerance for distance. The default is 1e-15.
+        radius_tol : float
+            Maximum allowed difference within radii for spherical distance
+            measures. The default is 1e-15.
```

Comment:
From this explanation, I don't understand what happens. Maybe a general explanation regarding tolerance could help. ... Or I#m just missing something.

Reply:
fixed